docs/user-guide.md

```diff
@@ -24,34 +25,50 @@ This guide provides detailed instructions on how to configure your Liberty proje
+
+    - [Liberty Maven Plugin latest release](https://github.com/OpenLiberty/ci.maven/releases/latest)
+
+    - [Liberty Gradle Plugin latest release](https://github.com/OpenLiberty/ci.gradle/releases/latest)
 
-  The following versions are recommended at minimum:
-  - Liberty Maven Plugin -> 3.7.1
-  - Liberty Gradle Plugin -> 3.5.1
+### Configure your Java
 
-### Settings
+Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server), the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta), the [XML Language Server](https://github.com/eclipse/lemminx) and start properly. A toast message alerts you if any language server fails to run or if Visual Studio Code has trouble locating your Java.
 
-Liberty Tools for Visual Studio Code requires Java 17 or later to ensure that the [Liberty Config Language Server](https://github.com/OpenLiberty/liberty-language-server) and the [Eclipse Language Server for Jakarta EE](https://github.com/eclipse/lsp4jakarta) start properly. A toast message alerts you if either language server fails to run.
+<img src="/docs/screenshots/update%20jdk%20toast.png" width="40%"/> 
+<img src="/docs/screenshots/java_17_toast_alert.png" width="40%" />
 
-<img src="/docs/screenshots/java_17_toast_alert.png" width="50%" height="50%">
+To resolve this issue, you can define properties in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings#_settingsjson) file or set system environment variables to point Liberty Tools to your Java 17 or later. 
 
-To resolve this conflict, define `java.jdt.ls.java.home` and `xml.java.home` in your Visual Studio Code [settings.json](https://code.visualstudio.com/docs/getstarted/settings) file to point Liberty Tools to your Java 17 or later runtime.
+For both Liberty Config Language Server and Eclipse Language Server for Jakarta EE, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
+1. The [embedded JRE](https://github.com/redhat-developer/vscode-java#java-tooling-jdk) included by [Language Support for Java(TM) by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.java). 
+  - By default, Liberty Tools installs the latest version of the rehat.java plugin, which contains an embedded JRE higher than Java 17 and therefore no configuration is required. However, if using an older version of the redhat.java plugin or using the universal version without the embedded JRE, use `java.jdt.ls.java.home` to specify your new JDK path.
+2. `JDK_HOME` or `JAVA_HOME` as system environment variables. (Note: if both `JDK_HOME` and `JAVA_HOME` are set, `JDK_HOME` will take precedence)
+
+For the XML Language Server, or LemMinX, Liberty Tools for Visual Studio Code will check for the Java versions in the following order:
```

Comment:
Will people know about the LemMinX component? Could we say "the language server supplied by the XML Language Support extension." Same comment for the mention of LemMinX in the table.

Reply:
<img width="802" alt="image" src="https://github.com/OpenLiberty/liberty-tools-vscode/assets/8934136/33f2b4a4-07a0-447b-8b8b-9e718f807ec8">

I've appended LemMinX onto the reference to the server here. Going forward, I think we can start referencing to it as LemMinX.

Reply:
Ok, that clarifies. Thanks.